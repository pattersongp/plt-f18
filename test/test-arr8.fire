func int main = () => {
        
        array[str, int] num;
        num["one"] = 7;
        num["other"] = 77;
        num["else"] = 11;

        print(num["one"]);
        print(num["other"]);
        print(num["else"]);

        int x = num["one"] * num["else"];
        print(x);

        return 0;
}