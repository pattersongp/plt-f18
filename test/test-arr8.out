7
77
11
77